onnxscript/values.py

```diff
@@ -204,7 +214,7 @@ def param_schemas_from_function_ir(
 
     schemas = []
     for arg in function_ir.inputs:
-        if isinstance(arg.typeinfo, onnx.TypeProto.Optional):
+        if _is_optional(arg.typeinfo):
```

Comment:
Is this a different type? The previous was checking for typeproto

Reply:
Very likely. Perhaps the previous one was typeinfo parsed from OpSchema?

Reply:
I see. So the previous check did nothing?

Reply:
This could need more fundamental fix around typeinfo. At least in this case it does not match `IRTypeLike` protocol. I'll open an issue.

Reply:
#775